src/document-converter.ts

```diff
@@ -573,6 +576,10 @@ export class DocumentConverter {
     removeUnnecessaryEventListeners(program);
     removeWrappingIIFEs(program);
     const importedReferences = this.collectNamespacedReferences(program);
+    const wasA11ySuiteAdded = addA11ySuiteIfUsed(
```

Comment:
Can `a11Suite` be treated as a namespaced reference and handled in `collectNamespacedReferences`?

Reply:
As discussed, it could but it would require a bit more work since a11ySuite is a bit of a one-off. Adding a todo to address later (if manifest conversion doesn't fix it for us.)